### DISCUSSION TITLE
What is the `.view` for on all the example links?

### DISCUSSION THREAD

**Discussion by lyssieth** [opensubsonic/open-subsonic-api#72] [General]
I'm working on my own implementation of a Subsonic and OpenSubsonic compatible server, and so far it's been going well.

I've got `Supersonic` (the desktop app, not the mobile app) working well, and I'm able to listen to music.

But now that I'm trying to ensure more apps work, like Supersonic on mobile (iOS), and Jamstash (web), I'm running into the `.view` extension on URLs a lot.
What's the purpose of it?

**Comment by Tolriq** [comment]
This is a legacy thing and while it should work without it, many clients will request it and some servers will only works with it.

So you should support .view at the end of the endpoint names.

**Reply by lyssieth** [reply]
Alright. For now I'm gonna just leave a janky `strip_suffix` in my caddy config while I figure out howw to support that on my server. Thanks for the immediate response.

Is it just meant to return exactly the same result? Or is there any oddities that I should know of?

**Reply by Tolriq** [reply]
It's the same like often you can call an url with .html or without at the end and both shows the same page.

**Reply by lyssieth** [reply]
Alright, gotcha. Thanks a lot!
